beta/lib/utils/index.js

```diff
@@ -0,0 +1,72 @@
+const path = require('path');
+const chalk = require('chalk');
+const rootPath = require('app-root-path');
+const latestVersion = require('latest-version');
+const pkg = require(path.join(rootPath.resolve(`/package.json`)));
+
+/**
+ * Checks to ensure that the generators/CLI in use are the latest ones
+ * If they are not show prompt to tell users to upgrade
+ * @param {*} pkgName
+ */
+async function checkGeneratorVersion(pkgName = null) {
+  let generatorPkg = pkg;
+
+  // if no pkgName then we assume this is for the CLI
+  if (pkgName) {
+    const packagePath = path.join(rootPath.resolve(`/node_modules/${pkgName}/package.json`));
```

Comment:
Same here with `require.resolve()`

Reply:
Added in a helper function to ensure package exists otherwise it will return `null`